tools/hive/hadoop-2.6.0/scripts/bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
+/usr/sbin/sshd
+: ${HADOOP_PREFIX:=/usr/local/hadoop}
+
+$HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+rm /tmp/*.pid
+
+# installing libraries if any - (resource urls added comma separated to the ACP system variable)
+cd $HADOOP_PREFIX/share/hadoop/common ; for cp in ${ACP//,/ }; do  echo == $cp; curl -LO $cp ; done; cd -
```

Comment:
We don't seem to use this functionality. We could consider just dropping it.

Reply:
dropped